tests/test_post_gen_project.py

```diff
@@ -0,0 +1,185 @@
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+from conftest import HOME
+
+sys.path.append(str(Path(__file__).resolve().parents[1]))
+from hooks.post_gen_project import (  # noqa: E402
+    copy_journal_template_files,
+    get_cookiecutter_dir,
+)
+
+
+@pytest.mark.parametrize(
+    "cookiecutters",
+    [
+        # test get_cookiecutter_dir when "scikit-package-manuscript"
+        #   cookiecutter exists.
+        # C1: with only "scikit-package-manuscript" cookiecutter.
+        ("scikit-package-manuscript",),
+        # C2: with "scikit-package-manuscript" and other cookiecutters.
```

Comment:
I actually think C1 is not needed.  I can't really think of a case where C2 passes but C1 doesn't.  It is ok to have it htere, but just a thought for the future.